DemoApp/controller/user.controller.js

```diff
@@ -44,8 +34,8 @@ const userRegister=async(req,res,employeetype)=>{
         }
 
         //create new user
-    const HashPassword=await bcrypt.hash(userDetails.password,12)
-    const NewUser=new User({
+        const HashPassword=await bcrypt.hash(userDetails.password,12)
+        const NewUser=new User({
```

Comment:
What you can do here to ommit line no. 50 is -> const NewUser = await new User({your object}).save();